### ISSUE TITLE
nwnsc not terminating after done compiling

### ISSUE THREAD

**Issue by StefanoND** [PhilippeChab/nwscript-ee-language-server#52]
First of all, thank you for this amazing extension!

I'm using Linux, Manjaro 6.2.10-1 and VSCodium with Marketplace and Features (from AUR)

I've been using your plugin for a while now and I've always noticed that as time goes on my computer fans starts ramping up and the entire system starts to slow down to a point where I was forced to reboot to fix this issue.

Then one day I decided to look into what was causing it, I ran "htop" command in the terminal to check for running processes and noticed that there was alot of nwnsc instances (called from your extension) running.

Since I'm too lazy I just created a script that runs "sudo killall nwnsc" periodically instead of posting the issue here.

So basically everytime the extension calls for nwnsc it never terminates it after its done.

Here's a screen snip of my hotp

![image](https://user-images.githubusercontent.com/45202623/232035274-bb4b375a-1c23-48da-8e22-15ddbb5d600a.png)

**Comment by PhilippeChab** [comment]
As we discussed on discord, it seems to be caused by the nwnsc itself - linux only. Closing this for now.
